alf/algorithms/actor_critic_algorithm.py

```diff
@@ -37,8 +35,10 @@ class ActorCriticAlgorithm(OnPolicyAlgorithm):
     def __init__(self,
                  observation_spec,
                  action_spec,
-                 actor_network: DistributionNetwork,
-                 value_network: Network,
+                 env,
+                 config,
```

Comment:
Since these two are optional. Do you think it's better to add `set_env()` and `set_config()` to RLAlgorithm?

Reply:
Changed ```env``` and ```config``` to optional to be consistent with RLAlgorithm currently. The current way looks good to me. Changing to ```set``` function does not provide too much advantage.